elisp/gilded-rose.el

```diff
@@ -0,0 +1,38 @@
+(defun make-item (name sell-in quality)
+  "Create an item with NAME, SELL-IN, and QUALITY."
+  (list :name name :sell-in sell-in :quality quality))
+
+(defun item-name (item)
+  "Return the name of ITEM."
+  (plist-get item :name))
+
+(defun item-sell-in (item)
+  "Return the sell-in value of ITEM."
+  (plist-get item :sell-in))
+
+(defun item-quality (item)
+  "Return the quality of ITEM."
+  (plist-get item :quality))
+
+(defun update-quality (item)
+  "Update the quality of the ITEM according to the Gilded Rose rules."
+  (let ((quality (item-quality item))
```

Comment:
the lets are already a simplification/improvement of the original code. Is it possible to access the value from the struct each time?

Reply:
Thank you. This was the simplest way I could represent the problem with the elisp I know. I'll have a look at getting the value from the struct each time